### PR TITLE
Add remaining CCMS MVP reports to database scripts

### PR DESCRIPTION
**Pull request by smallmanj-moj**
Add remaining report entries to the database scripts

### CODE REVIEW

File: src/main/resources/DDL_CreateGpfdTablesMojfin.sql [hunk 1]
```diff
@@ -55,5 +55,9 @@ Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REP
...
+Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (10,'CCMS General Ledger (tiny columns multiple periods manual)','SELECT * FROM ANY_REPORT.V_CCMS_GL_EXTRACTOR_MANUAL_YTD','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','CCMS General ledger extractor (tiny columns multiple periods manual)','CCMS_GEN_LEDGER_TINYCOL_MULPERIODS_MANUAL',10,'MAIN','William.Moran@Justice.gov.uk');
+Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (11,'CCMS Authorised expenditure (Contracting live system)','SELECT * FROM ANY_REPORT.V_CCMS_AUTHORISED_INVS_CONTRACTING','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','CCMS Authorised expenditure (Contracting live system)','CCMS_AUTH_EXPEND_CONTRACTING',11,'MAIN','William.Moran@Justice.gov.uk');
```
Comment (SineadCummings) [inline src/main/resources/DDL_CreateGpfdTablesMojfin.sql:61]: Are we able to condense down to a single INSERT INTO with multiple values? Same as data.sql file?

File: src/test/java/uk/gov/laa/gpfd/controller/ReportsControllerTest.java [hunk 2]
```diff
@@ -109,6 +109,11 @@ void getReportListReturnsCorrectResponseEntity() throws Exception {
 
     }
 
+    @Test
```
Comment (SineadCummings) [inline src/test/java/uk/gov/laa/gpfd/controller/ReportsControllerTest.java:112]: Has this been included intentionally? If so, what it's purpose?

**Review by SineadCummings — CHANGES_REQUESTED** [review]
Couple of minor comments

File: src/main/resources/DDL_CreateGpfdTablesMojfin.sql (hunk 1 above)
Comment (smallmanj-moj) [reply src/main/resources/DDL_CreateGpfdTablesMojfin.sql:61]: not with this version of ORACLE sadly.

**Review by SineadCummings — APPROVED** [review]

**Pull request closed without merging**